DataDefinitions/Passenger.cs

```diff
@@ -16,11 +16,11 @@ public class Passenger
 
         public Passenger(Passenger Passenger)
         {
-            this.missionid = missionid;
-            this.type = type;
-            this.vip = vip;
-            this.wanted = wanted;
-            this.amount = amount;
+            this.missionid = Passenger.missionid;
+            this.type = Passenger.type;
+            this.vip = Passenger.vip;
+            this.wanted = Passenger.wanted;
+            this.amount = Passenger.amount;
```

Comment:
This constructor appears to be entirely unused. Do we need the constructor at all?

If we need it, let's null protect here too.